aiida_common_workflows/workflows/relax/nwchem/generator.py

```diff
@@ -180,11 +203,10 @@ def get_builder(
 
         # Prepare builder
         builder = self.process_class.get_builder()
-
         builder.nwchem.code = orm.load_code(engines['relax']['code'])
         builder.nwchem.metadata.options = engines['relax']['options']
-        builder.nwchem.parameters = orm.Dict(dict=parameters)
-        builder.nwchem.add_cell = orm.Bool(True)
         builder.nwchem.structure = structure
+        builder.nwchem.add_cell = add_cell
```

Comment:
Just as a note: some plugins use a single `settings` `Dict` input node to have these kinds of "meta" parameters. Not saying you should change this or use that concept at all, just something you may want to consider if you think there will be a lot of small flags and switches that might accrue. At some point it may become easier to have them grouped in a single `settings` input.

Reply:
This is useful, thank you. I'll make that change in a future release.